test/box-luatest/gh_6920_crash_in_on_replace_triggers_test.lua

```diff
@@ -0,0 +1,26 @@
+local server = require('test.luatest_helpers.server')
+local t = require('luatest')
+local g = t.group()
+
+g.before_all = function()
+    g.server = server:new({alias = 'master'})
+    g.server:start()
+end
+
+g.after_all = function()
+    g.server:stop()
+end
+
+g.test_crash_in_case_of_ddl_ops_from_on_replace_triggers = function()
+    g.server:exec(function()
+        local t = require('luatest')
+        local s = box.schema.space.create('test')
+        local _ = s:create_index('pk')
+        s:on_replace(function() end)
+        s:on_replace(function () s:format({{'a', 'unsigned'}}) end)
+        t.assert_error_msg_content_equals(
+            "Space on_replace triggers does not support ddl operations",
+            function() return s:replace({1}) end
+        )
+    end)
+end
```

Comment:
I don't think we need this test - it's already checked by the existing tests you update in this patch. Let's drop it.

Reply:
fixed